channels/fi.m3u

```diff
@@ -1,13 +1,29 @@
-#EXTINF:-1 tvg-id="TaivasTV7.fi" tvg-country="FI" tvg-language="" tvg-logo="https://i.imgur.com/pIYSEUN.jpg" group-title="",Taivas TV7 (720p)
-http://vod.tv7.fi:1935/tv7-fi/tv7-fi.stream_720p/chunklist_w2131949141.m3u8
-#EXTINF:-1 tvg-id="TV7.fi" tvg-country="FI;SE;EE" tvg-language="Finnish" tvg-logo="https://i.imgur.com/pIYSEUN.jpg" group-title="",TV7 (720p)
-http://vod.tv7.fi:1935/tv7-fi/tv7-fi.stream_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="TaevasTV7.fi" tvg-country="EE" tvg-language="Finnish;Estonian" tvg-logo="https://i.imgur.com/pIYSEUN.jpg" group-title="Religious",Taevas TV7 (720p)
+https://vod.tv7.fi/tv7-ee/tv7-ee.stream_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="TaivasTV7.fi" tvg-country="FI" tvg-language="Finnish" tvg-logo="https://i.imgur.com/pIYSEUN.jpg" group-title="Religious",Taivas TV7 (720p)
+https://vod.tv7.fi/tv7-fi/tv7-fi.stream_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="NebesaTV7.fi" tvg-country="RU" tvg-language="Russian" tvg-logo="https://www.nebesatv7.com/wp-content/themes/tv7-theme/assets/img/logo_nebesa_short.png" group-title="Religious",Небеса ТВ7 (720p)
+https://vod.tv7.fi/tv7-ru/_definst_/smil:tv7-ru.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="NebesaTV7.fi" tvg-country="RU" tvg-language="Russian" tvg-logo="https://www.nebesatv7.com/wp-content/themes/tv7-theme/assets/img/logo_nebesa_short.png" group-title="Religious",Небеса ТВ7 (720p)
+https://vod.tv7.fi/tv7-ru/tv7-ru.stream_720p/playlist.m3u8
+#EXTINF:0 tvg-id="YLETeemaFem.fi" tvg-country="INT" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/Yle_Teema_%26_Fem.svg/799px-Yle_Teema_%26_Fem.svg.png" group-title="",YLE Teema Fem (720p)
+https://yletvhdliveworld-lh.akamaihd.net/i/yleteemafemfiw_1@308915/master.m3u8
+#EXTINF:0 tvg-id="YLETeemaFem.fi" tvg-country="FI" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/Yle_Teema_%26_Fem.svg/799px-Yle_Teema_%26_Fem.svg.png" group-title="",YLE Teema Fem (720p) [Geo-blocked]
+https://yletv-lh.akamaihd.net/i/yleteemafemfi_1@490775/master.m3u8
+#EXTINF:0 tvg-id="YLETV1.fi" tvg-country="INT" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Yle_TV1_logo.svg/382px-Yle_TV1_logo.svg.png" group-title="",YLE TV 1 (720p)
+https://yletvhdliveworld-lh.akamaihd.net/i/yletv1hdworld_1@187592/master.m3u8
+#EXTINF:0 tvg-id="YLETV1.fi" tvg-country="FI" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Yle_TV1_logo.svg/382px-Yle_TV1_logo.svg.png" group-title="",YLE TV 1 (720p) [Geo-blocked]
+https://yletv-lh.akamaihd.net/i/yletv1hls_1@103188/master.m3u8
+#EXTINF:0 tvg-id="YLETV2.fi" tvg-country="INT" tvg-language="Finnish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Yle_TV2_logo.svg/399px-Yle_TV2_logo.svg.png" group-title="",YLE TV 2 (720p)
+https://yletvhdliveworld-lh.akamaihd.net/i/yletv2hdworld_1@187593/master.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/5430116/133936520-e66f9368-9c54-4c71-869b-43f42c4255c3.png)
May have a sense to mark global source as Not 24/7 since some programs can only be viewed in Finland